### ISSUE TITLE
Shap explainer plot "stuck"/broken, incorrect output value

### ISSUE THREAD

**Issue by Ido-PX** [shap/shap#442]
I've been using the package for quite a while now and it has helped quite a lot, so thank you.

I've run into an issue where seemingly no matter what I do, the shap explainer plot is the same, displaying a wrong output value and is missing any indication of features. 

Attempted to restart kernel and re-run all the code to no avail, I'm not getting any errors and couldn't find a similar issue here.

I'm using it to explain XGBoost predictions. 

Here's what it looks like:
![screen shot 2019-02-12 at 10 48 15](https://user-images.githubusercontent.com/42135371/52623314-ae040200-2eb4-11e9-818a-b09fdadf0d37.png)
In the plot shown in the picture the model prediction should actually be 0.001

and my code:
(man_test_df_X_features is a pandas series of features.)
```import shap

# load JS visualization code to notebook
shap.initjs()

# explain the model's predictions using SHAP values
explainer = shap.TreeExplainer(xgb_model)

shap_values = explainer.shap_values(man_test_df_X_features.values)
shap.force_plot(explainer.expected_value, shap_values, man_test_df_X_features,  link= 'logit')```

**Comment by Ido-PX** [comment]
I've created a new environment, reinstalled all the relevant packages and still get exactly the same output.

**Comment by slundberg** [comment]
That's odd. Can you print and share the values in `shap_values`? You might also try using `man_test_df_X_features.values` in the force_plot call.

**Comment by Ido-PX** [comment]
@slundberg thank you. I've figured it out and I'll answer my own question: 

This issue likely ensued when I did conda update all. The packages were updated, however I was using shap to explain an xgboost model trained from an older version of xgboost (I pickled the model a while back).

I'm not sure what the issue was, but I've created a new model with the latest xgboost library and all seems to be well, so I assume that was the issue.
